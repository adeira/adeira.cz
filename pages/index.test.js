// @flow

import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Index from './index';

const renderer = new ShallowRenderer();

it('Works!', () => {
  expect(renderer.render(<Index />)).toMatchSnapshot();
});
