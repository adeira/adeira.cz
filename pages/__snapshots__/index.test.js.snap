// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works! 1`] = `
<span>
  ðŸ’©
</span>
`;
